synapse/python_dependencies.py

```diff
@@ -95,7 +95,7 @@
     "oidc": ["authlib>=0.14.0"],
     "systemd": ["systemd-python>=231"],
     "url_preview": ["lxml>=3.5.0"],
-    "test": ["mock>=2.0", "parameterized"],
+    "test": ["mock>=2.0", "parameterized>=0.7.0"],
```

Comment:
generally we try to include a comment about why we need a particular version, to help distinguish "this is what we tested against, others may work" from "we know others won't work".